app/waf/src/models/pool.model.ts

```diff
@@ -30,9 +25,82 @@ export class Pool extends Entity {
     itemType: 'string',
     required: false,
     default: [],
+    schema: {
+      create: true,
+      update: true,
+      response: true,
+      example: [
+        '23442d6a-4d82-40ee-8d08-243750dbc191',
+        'aaaabbda-4d82-40ee-8d08-243750dbc192',
+      ],
+    },
   })
   monitors: string[];
 
+  @property({
+    type: 'number',
+    required: false,
+    default: 1,
+    schema: {
+      create: true,
+      update: true,
+      response: true,
+      example: 1,
+    },
+  })
+  minimumMembersActive: number;
+
+  @property({
+    type: 'number',
+    required: false,
```

Comment:
Its default value should be 0 or 1?

Reply:
I refer the others default value is `1`.
 
and I check the description: `Member is down when fewer than minimum monitors report it healthy`

It is reasonable to be `1`